packages/shared/components/popups/SendConfirmationPopup.svelte

```diff
@@ -23,8 +21,8 @@
 
         function _send(): void {
             return recipient.type === 'account'
-                ? sendInternalTransaction($selectedAccount.id, recipient.account.depositAddress, rawAmount, internal)
-                : sendExternalTransaction($selectedAccount.id, recipient.address, rawAmount)
+                ? void trySendAmount(recipient.account.depositAddress, rawAmount)
+                : void trySendAmount(recipient.address, rawAmount)
         }
```

Comment:
`return void trySendAmount(recipient.type === 'account' ? recipient.account.depositAddress : recipient.address, rawAmount)`

Reply:
Or: `const recipientAddress = recipient.type === 'account' ? recipient.account.depositAddress : recipient.address` and on a new line the trySendAmount function.